lib/matplotlib/backends/backend_ps.py

```diff
@@ -610,9 +610,9 @@ def draw_path_collection(self, gc, master_transform, paths, all_transforms,
             path_codes.append(name)
 
         for xo, yo, path_id, gc0, rgbFace in self._iter_collection(
-            gc, master_transform, all_transforms, path_codes, offsets,
-            offsetTrans, facecolors, edgecolors, linewidths, linestyles,
-            antialiaseds, urls, offset_position):
+                gc, master_transform, all_transforms, paths, path_codes,
+                offsets, offsetTrans, facecolors, edgecolors, linewidths,
+                linestyles, antialiaseds, urls, offset_position):
```

Comment:
Yeah -- just a "fixing while I'm in there adding arguments anyway" kind of thing.